ansible/roles/ssh/tasks/main.yaml

```diff
@@ -22,5 +22,14 @@
     - { src: "{{jenkins_public_ssh_key_file_path}}",
         dest: "{{jenkins_home_dir}}/.ssh/authorized_keys",
         owner: jenkins, group: jenkins, mode: "0600" }
+
+- name: Add Github and upload server's host key
```

Comment:
So you suggest that I change to "Add known keys for remote hosts" in both prompt and task name, then be more specific in the Ansible `README` file about the content of `known_hosts`?